theano/sandbox/multinomial.py

```diff
@@ -239,6 +250,7 @@ def c_code_cache_version(self):
     def c_code(self, node, name, ins, outs, sub):
         (pvals, unis, n) = ins
         (z,) = outs
+        replace = ("1" if self.replace else "0")
```

Comment:
as self.replace is a bool, you can simplify to "int(self.replace)" and use "%(replace)d" in the c code.